translations/client_de.ts

```diff
@@ -624,7 +624,7 @@ Wählen Sie diejenigen aus, die Sie synchronisieren möchten:</translation>
     <message>
         <location filename="../src/gui/clientgui.cpp" line="1196"/>
         <source>REMOVE ALL SYNCHRONIZATIONS</source>
-        <translation>ALLE SYNCHRONISATIONEN ENTFERNEN</translation>
+        <translation>ALLE SYNC. ENTFERNEN</translation>
```

Comment:
I am hesitant: "ALL SYNCS ENTFERNEN" seems to be another viable choice.

Reply:
I don't speak Deutsch well, but DeepL and ChatGPT both seem to suggest:
> ALL SYNCS ENTFERNEN

Reply:
```suggestion
        <translation>ALLE SYNCS ENTFERNEN</translation>
```

Reply:
Sounds like a solid argument in favor of "Sync." 👍